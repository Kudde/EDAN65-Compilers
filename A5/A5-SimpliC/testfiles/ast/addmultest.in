int addmul() {
  int x;
  int y;
  int k;
  k = x * y + 3;
}
