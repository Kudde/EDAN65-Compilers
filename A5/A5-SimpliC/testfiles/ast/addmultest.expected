Program
  List
    Fun
      IntType ID="int"
      IdDecl ID="addmul"
      Param
        List
      Block
        List
          DeclStmt
            IntType ID="int"
            IdDecl ID="x"
            Opt
          DeclStmt
            IntType ID="int"
            IdDecl ID="y"
            Opt
          DeclStmt
            IntType ID="int"
            IdDecl ID="k"
            Opt
          AssignStmt
            IdUse ID="k"
            Plus
              Mul
                IdUse ID="x"
                IdUse ID="y"
              Literal NUMERAL="3"
